keps/sig-network/1860-kube-proxy-IP-node-binding/README.md

```diff
@@ -149,3 +159,284 @@ On downgrade, the feature gate will simply be disabled, and as long as `kube-pro
 ### Version Skew Strategy
 
 Version skew from the control plane to `kube-proxy` should be trivial since `kube-proxy` will simply ignore the `ipMode` field.
+
```

Comment:
Since I can't comment on unchanged code, commenting here:

For Upgrade/Downgrade section:

"all the previous LoadBalancer service will get an `ipMode` of `VIP`."

Will cloud-controller-manager be explicitly setting those?
Or are we going to rely on this being defaulted to VIP on the first write by kube-apiserver?

Reply:
It will be defaulted to VIP when we get it from kube-apiserver in this case. Ref https://github.com/kubernetes/kubernetes/pull/118895/files#r1248316868

Reply:
Can you add that explicitly there (with the link - it seems helpful here too).

Reply:
Added.